app/serializers/page_category_serializer.rb

```diff
@@ -0,0 +1,21 @@
+class PageCategorySerializer < ActiveModel::Serializer
+  attributes :id, :type, :title, :description
+
+  private
+
+  def id
+    object.slug
+  end
+
+  def title
+    object.label
+  end
+
+  def description
+    object.meta_description
+  end
+
+  def type
+    'guide'
```

Comment:
Two fold: 
1) to keep it consistent with the existing API. 
2) the frontend depends on this being here. 

I think you are right in the sense that it is an old name from hippo, that should not really be carried forward. But I think we need to do it in two stages, firstly make sure everything works, then deprecate the category service and then we can remove this, or name it more in line with the domain.